### DISCUSSION TITLE
How to enable re-order of items, without allowing them to be dropped onto other lists

### DISCUSSION THREAD

**Discussion by professordave** [punker76/gong-wpf-dragdrop#434] [Q&A] [answered]
Are there any settings that enable the following?

There are three listviews on the screen, that just happen to have the same object type, so when all options are enabled, can drag and drop items between the 3 lists..

Suppose we want to allow re-ordering within a list, but do NOT want items to be dragged between lists? 

Is there an easy way to do this?

**Comment by punker76** [comment, marked as answer]
@professordave Yes, this is possible with the ‘DragDropContext’ property. If you set this for every list with different key then you can only drag&drop inside this context. ([All properties](https://github.com/punker76/gong-wpf-dragdrop/wiki/Properties))

**Reply by professordave** [reply]
Thanks, for the quick response! 

DragDropContext enables this to work!

This works, just tested.
